examples/main.cpp

```diff
@@ -38,55 +41,46 @@ using namespace tiny_dnn::layers;
+void sample1_convnet(const string &data_dir = "../data");
+void sample2_mlp(const string &data_dir = "../data");
 void sample3_dae();
-void sample4_dropout(const string& data_dir = "../data");
-void sample5_unbalanced_training_data(const string& data_dir = "../data");
+void sample4_dropout(const string &data_dir = "../data");
+void sample5_unbalanced_training_data(const string &data_dir = "../data");
 void sample6_graph();
 
-int main(int argc, char** argv) {
+int main(int argc, char **argv) {
     try {
         if (argc == 2) {
             sample1_convnet(argv[1]);
         } else {
             sample1_convnet();
         }
-    }
-    catch (const nn_error& e) {
+    } catch (const nn_error &e) {
         std::cout << e.what() << std::endl;
     }
 }
 
 ///////////////////////////////////////////////////////////////////////////////
 // learning convolutional neural networks (LeNet-5 like architecture)
-void sample1_convnet(const string& data_dir) {
+void sample1_convnet(const string &data_dir) {
     // construct LeNet-5 architecture
     network<sequential> nn;
-    adagrad optimizer;
+    adagrad             optimizer;
 
-    // connection table [Y.Lecun, 1998 Table.1]
+// connection table [Y.Lecun, 1998 Table.1]
 #define O true
 #define X false
-    static const bool connection[] = {
-        O, X, X, X, O, O, O, X, X, O, O, O, O, X, O, O,
-        O, O, X, X, X, O, O, O, X, X, O, O, O, O, X, O,
-        O, O, O, X, X, X, O, O, O, X, X, O, X, O, O, O,
-        X, O, O, O, X, X, O, O, O, O, X, X, O, X, O, O,
-        X, X, O, O, O, X, X, O, O, O, O, X, O, O, X, O,
-        X, X, X, O, O, O, X, X, O, O, O, O, X, O, O, O
-    };
+    static const bool connection[] = {O, X, X, X, O, O, O, X, X, O, O, O, O, X, O, O, O, O, X, X, X, O, O, O,
+                                      X, X, O, O, O, O, X, O, O, O, O, X, X, X, O, O, O, X, X, O, X, O, O, O,
+                                      X, O, O, O, X, X, O, O, O, O, X, X, O, X, O, O, X, X, O, O, O, X, X, O,
+                                      O, O, O, X, O, O, X, O, X, X, X, O, O, O, X, X, O, O, O, O, X, O, O, O};
 #undef O
 #undef X
 
-    nn << convolutional_layer<tan_h>(
-            32, 32, 5, 1, 6)  /* 32x32 in, 5x5 kernel, 1-6 fmaps conv */
-       << average_pooling_layer<tan_h>(
-            28, 28, 6, 2)     /* 28x28 in, 6 fmaps, 2x2 subsampling */
-       << convolutional_layer<tan_h>(
-            14, 14, 5, 6, 16, connection_table(connection, 6, 16))
-       << average_pooling_layer<tan_h>(10, 10, 16, 2)
-       << convolutional_layer<tan_h>(5, 5, 5, 16, 120)
+    nn << convolutional_layer<tan_h>(32, 32, 5, 1, 6) /* 32x32 in, 5x5 kernel, 1-6 fmaps conv */
+       << average_pooling_layer<tan_h>(28, 28, 6, 2)  /* 28x28 in, 6 fmaps, 2x2 subsampling */
+       << convolutional_layer<tan_h>(14, 14, 5, 6, 16, connection_table(connection, 6, 16))
+       << average_pooling_layer<tan_h>(10, 10, 16, 2) << convolutional_layer<tan_h>(5, 5, 5, 16, 120)
```

Comment:
Please add a line break between `average_pooling_layer<tan_h>(10, 10, 16, 2)` and `convolutional_layer<tan_h>(5, 5, 5, 16, 120)`.